openpype/plugins/publish/validate_editorial_asset_name.py

```diff
@@ -0,0 +1,112 @@
+import pyblish.api
+from avalon import io
+from pprint import pformat
+
+
+class ValidateEditorialAssetName(pyblish.api.ContextPlugin):
+    """ Validating if editorial's asset names are not already created in db.
+
+    Checking variations of names with different size of caps or with
+    or without underscores.
+    """
+
+    order = pyblish.api.ValidatorOrder
+    label = "Validate Asset Name"
+
+    def process(self, context):
+
+        asset_and_parents = self.get_parents(context)
+
+        if not io.Session:
+            io.install()
+
+        db_assets = list(io.find(
+            {"type": "asset"}, {"name": 1, "data.parents": 1}))
+        self.log.debug("__ db_assets: {}".format(db_assets))
+
+        project_entities = {
```

Comment:
Just a note not change requirement as it's on multiple places. They are docs, not entities. Entitiy would be more complex object (as I found out recently).

Reply:
thanks for suggestion. Changed.